packages/compass-e2e-tests/helpers/commands/connect-form.ts

```diff
@@ -319,8 +317,8 @@ async function getCheckedRadioValue(
   browser: CompassBrowser,
   selector: string
 ): Promise<string | null> {
-  const elements = await browser.$$(selector);
-  for (const element of elements) {
+  const elements = browser.$$(selector);
+  for await (const element of elements) {
```

Comment:
browser.$$ returns a chainable element promise array so a) it isn't a promise itself anymore and b) we have to async iterate over the result.